bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/AssetIngestor.java

```diff
@@ -297,7 +299,6 @@ private void createAsset(Source source, String assetPath, ResourceResolver r, bo
 
                 if (asset == null) {
                     AssetIngestorException ex = new AssetIngestorException("Cannot create asset: asset is null on path  " + assetPath);
-                    Logger.getLogger(AssetIngestor.class.getName()).log(Level.SEVERE, null, ex);
```

Comment:
We should log this properly too, I think.

Reply:
You assume that this exception is not logged somewhere else? I hate huge stacktraces, which appear a few times in log files, because everyone is logging and throwing them. You end up with 100kbytes of log for just 1 error :-)

Reply:
Assume?  Who do you think wrote the framework for this one? ;P